src/users.ts

```diff
@@ -122,14 +122,21 @@ function getTeam(team: string, org: string, teams: Team[]) {
   });
 }
 
-export async function reconcileUsers() {
-  const promises = new Array<Promise<Octokit.AnyResponse | void>>();
+export async function reconcileUsers(reconcileAdmin = false) {
   const teams = await reconcileTeams();
+  await _reconcileUsers(teams);
+  await _reconcileUsers(teams, true);
+}
+
+async function _reconcileUsers(teams: Team[], reconcileAdmins = false) {
+  const promises = new Array<Promise<Octokit.AnyResponse | void>>();
   for (const o of users.orgs) {
     for (const m of users.membership) {
-      const team = getTeam(m.team, o, teams);
+      const teamName = reconcileAdmins ? `${m.team}-admins` : m.team;
+      const team = getTeam(teamName, o, teams);
+      const users = reconcileAdmins ? m.admins : m.users;
```

Comment:
This function is 60 lines of code, and with the new line 137, it contains 8 branching statements.  As a consequence, I find it hard to follow.  Could you break it into 2 or more smaller functions?